code/modules/mob/living/simple_animal/hostile/terror_spiders/queen.dm

```diff
@@ -0,0 +1,434 @@
+		if(ckey)
+			madnesstarget = null
+			madnesstarget = input(src,"Which person should fear?") in null|choices
+		if(!madnesstarget)
+			// cancel
+		else
+			var/mob/living/carbon/human/H = madnesstarget
+			H.hallucination = max(H.hallucination,300)
+			to_chat(H,"<span class='danger'>Your head throbs in pain.</span>")
+			to_chat(src, "You reach through bluespace into the mind of [madnesstarget], making their fears come to life. They start to hallucinate.")
+	else
+		to_chat(src, "You have run out of uses of this ability.")
+
+/mob/living/simple_animal/hostile/poison/terror_spider/queen/verb/VerbQueenScreech()
+	set name = "Queen Screech"
+	set category = "Spider"
+	set desc = "Emit horrendusly loud screech which breaks all nearby cameras and most nearby lights. Can only be used twice!"
+	if(spider_can_screech)
+		spider_can_screech--
+		DoQueenScreech(15,50,10,100)
+	else
+		to_chat(src, "You have run out of uses of this ability.")
+
+
+/mob/living/simple_animal/hostile/poison/terror_spider/queen/proc/DoQueenScreech(var/light_range, var/light_chance, var/camera_range, var/camera_chance)
+	visible_message("<span class='userdanger'>\The [src] emits a bone-chilling shriek!</span>")
+	for(var/obj/machinery/light/L in orange(light_range,src))
+		if(L.on && prob(light_chance))
+			L.broken()
+	for(var/obj/machinery/camera/C in orange(camera_range,src))
+		if(C.status && prob(camera_chance))
+			C.toggle_cam(src,0)
+
+
+/mob/living/simple_animal/hostile/poison/terror_spider/queen/verb/QueenFakeLings()
+	set name = "Fake Spiderlings"
+	set category = "Spider"
+	set desc = "Animates some damaged spiderlings to crawl throughout the station and panic the crew. Sows fear. These spiderlings never mature. Ability can only be used 3 times."
+	if(spider_can_fakelings)
+		spider_can_fakelings--
+		var/numlings = 15
+		for(var/i in 1 to numlings)
+			var/obj/effect/spider/spiderling/terror_spiderling/S = new /obj/effect/spider/spiderling/terror_spiderling(get_turf(src))
+			S.grow_as = /mob/living/simple_animal/hostile/poison/terror_spider/red
+			S.stillborn = 1
+			S.name = "Evil-Looking Spiderling"
+			S.desc = "It moves very quickly, hisses loudly for its size... and has disproportionately large fangs. Hopefully it does not grow up..."
+	else
+		to_chat(src, "You have run out of uses of this ability.")
+
+
+/mob/living/simple_animal/hostile/poison/terror_spider/queen/ShowGuide()
+	..()
+	to_chat(src, "QUEEN OF TERROR guide:")
+	to_chat(src, "- Your goal is to build a nest, Lay Eggs to make more spiders, and ultimately exterminate all non-spider life on the station!")
+	to_chat(src, "- You can use HiveCommand to issue orders to your spiders (both AI, and player-controlled)!")
+	to_chat(src, "- The success or failure of the entire hive depends on you, so whatever you do, <b>do not die!</b>")
+	to_chat(src, "- To start, find a safe, dark location to nest in, then lay some purple (praetorian) eggs so you will have guards to defend you.")
+	to_chat(src, " ")
```

Comment:
do not send empty messages what the fuck
send breaks in messages if you have to